### DISCUSSION TITLE
Release 2.11.0

### DISCUSSION THREAD

**Discussion by cedricbonhomme** [monarc-project/MonarcAppFO#375] [General]
### New

- having the possibility to define custom scales for operational risks ([#353](https://github.com/monarc-project/MonarcAppFO/issues/353));
- introduction of the risk context and the risk owner ([#21](https://github.com/monarc-project/MonarcAppFO/issues/21), [#186](https://github.com/monarc-project/MonarcAppFO/issues/186)).

### Fix

- update-all.sh: Could not read from remote repository ([#365](https://github.com/monarc-project/MonarcAppFO/issues/365));
- some files in script do not have the correct permissions ([#364](https://github.com/monarc-project/MonarcAppFO/issues/364)).

![Screenshot_20210902_133851](https://user-images.githubusercontent.com/465400/131843376-d9b7420c-0c9f-430e-8269-42ef3478dbf2.png)

![Screenshot_20210902_134102](https://user-images.githubusercontent.com/465400/131843404-3cad5d50-406d-431c-84a7-4e7b80460ad9.png)

![Screenshot_20210902_142720](https://user-images.githubusercontent.com/465400/131843437-4ff5953d-3e5b-42bd-94cb-012baf2e868f.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/monarc-project/MonarcAppFO/releases/tag/v2.11.0'>Release 2.11.0</a>.</em>

**Comment by K8Sudo** [comment]
@cedricbonhomme The new fields added with the update are currently not yet available at translate.monarc.lu. This means that we cannot yet translate terms such as "Risk owner".

Best regards

Marc

**Reply by cedricbonhomme** [reply]
Thank you for the information. We are updating the internationalization branch.
